kuma/static/js/wiki-compat-tables.js

```diff
@@ -53,8 +55,10 @@
                     'aria-expanded': false,
                     'aria-controls': historyId
                 });
-                $td.find('.bc-history').attr('id', historyId);
-                $td.find('.bc-history-link').attr('tabIndex', -1);
+                var $history = $td.find('.bc-history');
```

Comment:
You may want to move this new line to right after `var $td = $(this);` since JavaScript vars are hoisted up.